### PR TITLE
Update HTML

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Update page layout by improving divs and links and images

[Finishes: #118669707]

### CODE REVIEW

**Comment by michaelwehrley** [comment]
Some some home page layout updates.

File: index.html [hunk 1]
```diff
@@ -21,17 +21,6 @@
...
-    .left-div{text-align: left;}
-    .left-div h1,h2,h3 {color:#520063;}
-    h4 {color:white;}
-    #login {margin-top: 4em;padding:2em; background-color: #520063; border-radius: 1em;}
-    #login h2 {color:white !important; margin:0; margin-bottom:.5em;}
-    .responseBtn {font-size:1.4em; text-align: left;}
-    label {text-align: left;}
-    .question {font-size: 1.5em; font-weight: bold;}
-    .btn {text-align: left;}
     img#inform-me {
```
Comment (eschlange) [inline index.html]: Can this be extracted to a common CSS file?

Comment (michaelwehrley) [inline index.html]: I will check.  Thanks!

**Comment by eschlange** [comment]
Other than the CSS comment, the content looks alright. Merge away.

**Comment by michaelwehrley** [comment]
🍦

**Pull request merged**